web/components/application/hacker/Hacker.test.js

```diff
@@ -246,7 +246,7 @@ describe('Hacker component', () => {
         });
 
         it('page one is rendered', () => {
-          expect(wrapper.find('div#hacker-application-page-1')).toHaveLength(1);
+          expect(wrapper.find('form#hacker-application-page-1')).toHaveLength(1);
```

Comment:
@mingyokim an example of the potentially-overly-specific-tests i talked about the other day

Reply:
yeah, i get it now. i'll try to avoid these patterns next time